src/inputs/DateFields/DateFieldBase.test.tsx

```diff
@@ -34,6 +34,24 @@ describe(DateFieldBase, () => {
     expect(r.date_helperText().textContent).toBe("Helper text");
   });
 
+  it("renders correctly without calendar icon button", async () => {
+    // Given a DateField without a calendar icon
+    const r = await render(
+        <DateFieldBase
```

Comment:
Nit: Run prettier? This looks 4-space indented and I think prettier will nudge it down to 2-space indented.

Reply:
I appreciate the nitpick! Fixed.